build-system/tasks/release/index.js

```diff
@@ -463,4 +484,7 @@ release.flags = {
     '  Directory path to emplace release files (defaults to "./release")',
   'flavor':
     '  Limit this release build to a single flavor. Can be used to split the release work between multiple build machines.',
+  'skip_commits_verification':
+    '  Skip verifying that every commit exists on master (or is a cherry-pick of a commit on master). If this is set, then you must provide version_override.',
+  'version_override': '  Overrides the version written to AMP_CONFIG',
```

Comment:
I wonder if it isn't enough to just have this flag, and discard the other flag. In either case we'll have to change the release automation backend to support this and to add a field to add the overridden version number for these releases, so I wouldn't want to merge this PR before the backend supports it

Reply:
Here's another solution that doesn't require `version_override` or backend changes - https://github.com/ampproject/amphtml/pull/32689

Reply:
@estherkim your PR seems fine by me (modulo the comment I left there, which is definitely not a blocker) but I think this current PR could still be relevant